doc/dev/osd_internals/manifest.rst

```diff
@@ -224,6 +347,8 @@ Operations:
 
   TODO: No user_modify means does not this trigger a new clone?
 
+  --> Yes, no user_modify + add_flag like cache_evict as described above.
```

Comment:
That's fine.  Remove the TODO and replace it with something like "This operation is not a user mutation and does not trigger a clone to be created".